src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -395,16 +395,21 @@ def _get_platform_info():
     return platform_name.lower(), release.lower()
 
 
-def _is_wsl(platform_name, release):
+def is_wsl():
```

Comment:
I'm wondering why the original  _is_wsl() is ignoring the passed in perameter `platform_name, release` and calls _get_platform_info() to overwrite them?

Reply:
Seems to be some forgotten logic.